### PR TITLE
Refactor Navbar

### LINKED ISSUE #369: Refactor UserNavbar UI
Looking at other navbars across the internet, we see:

Github:
image

NFL:
image

Chocolate Milk:
image

We see a common pattern where content is *split* between the left and right sides. Now let's look at our navbar:
image

I don't know about you, but the blue, plain font and two grey icons is disgusting. Let's fix it!

### The move
1. Realize that the code for the navbar is in [`src/Components/Navbar/UserNavbar.js`](https://github.com/SCE-Development/Core-v4/blob/dev/src/Components/Navbar/UserNavbar.js).
1. Move website links to the left and keep account drop down on right.
1. Remove icons to LinkedIn and Facebook and "Student Resources" dropdown
1. Add style! Consider making the color a dark blue like our shirts.
1. Github only has their logo on the left side (see above). Add a small SCE icon to the very left of the navbar to replace the text.

### PR DESCRIPTION
**Pull request by nanarboursal**
Resolves #369 

Screen Shot 2020-05-12 at 2 48 23 PM

### CODE REVIEW

File: src/Components/Navbar/UserNavbar.js [hunk 1]
```diff
@@ -13,71 +13,40 @@ import {
...
-        '3,19V5 A2,2 0 0,1 5,3M18,5H15.5A3.5,3.5 0 0,0 12,8.5V11H10V14H1',
-        '2V21H15V14H18V11H15V9A1,1 0 0,1 16,8H18V5Z'
-      ].join('')
-    }
-  ];
   const unauthedRoutes = [{ title: 'Events', route: '/events' }];
 
   return (
     <div className='user-nav'>
+
```
Comment (seemavora) [inline src/Components/Navbar/UserNavbar.js:24]: remove space

**Comment by ThaiQ** [comment]
@nanarboursal work with @surabhigupta2029  for a new design 
https://www.youtube.com/watch?v=PwWHL3RyQgk
https://www.youtube.com/watch?v=biOMz4puGt8&t=518s
https://www.youtube.com/watch?v=EyeUrJtUCic

File: src/Components/Navbar/UserNavbar.js [hunk 2]
```diff
@@ -13,71 +13,40 @@ import {
...
   return (
     <div className='user-nav'>
       <Navbar light expand='md'>
         <NavbarBrand href='/'>
-          Software & Computer Engineering Society
+          <div>
+            <img
+              src={logo}
+              alt={'sce logo'}
+              style={{
```
Comment (evanugarte) [inline src/Components/Navbar/UserNavbar.js:30]: consider not using inline styles and moving this to `.css`

File: src/Components/Navbar/UserNavbar.js [hunk 3]
```diff
@@ -91,25 +60,18 @@ export default function UserNavBar(props) {
             {unauthedRoutes.map((link, index) => {
               return (
                 <NavItem key={index}>
-                  <NavLink href={link.route}>{link.title}</NavLink>
+                  <NavLink
+                    style={{color: 'white', fontWeight: 'bold'}}
```
Comment (evanugarte) [inline src/Components/Navbar/UserNavbar.js:64]: same here, add a `className` and do it in `.css`

File: src/Components/Navbar/UserNavbar.js [hunk 4]
```diff
@@ -91,25 +60,18 @@ export default function UserNavBar(props) {
...
-                  <DropdownItem href='/register'>
-                    Membership Application
-                  </DropdownItem>
-                </DropdownMenu>
-              </UncontrolledDropdown>
-            )}
             {props.authenticated && props.user ? (
               <UncontrolledDropdown nav inNavbar>
-                <DropdownToggle nav caret>
+                <DropdownToggle nav caret style={{color: 'white'}}>
```
Comment (evanugarte) [inline src/Components/Navbar/UserNavbar.js:74]: no inline!!!

File: src/Components/Navbar/UserNavbar.js [hunk 5]
```diff
@@ -146,7 +108,21 @@ export default function UserNavBar(props) {
                 </DropdownMenu>
               </UncontrolledDropdown>
             ) : (
-              <NavLink href='/login'>Login</NavLink>
+              <UncontrolledDropdown nav inNavbar>
+                <DropdownToggle nav caret
+                  style={{color: 'white', marginRight: 10, fontWeight: 'bold'}}
```
Comment (evanugarte) [inline src/Components/Navbar/UserNavbar.js:113]: move to css

**Comment by nanarboursal** [comment]
Screen Shot 2020-06-06 at 5 47 16 PM

File: src/Components/Navbar/navbar.css [hunk 6]
```diff
@@ -21,20 +21,47 @@
...
+
+.logout-image {
+  width: 18px;
+  height: 18px;
+}
+
+.profile-image {
+  width: 30px;
+  height: 30px;
+}
```
Comment (IsitaB) [inline src/Components/Navbar/navbar.css:67]: blank line at the end of the file

**Comment by nanarboursal** [comment]
Screen Shot 2020-06-28 at 4 33 33 PM
here's what the button dropdown looks like when the page is condensed. the other menu shows up when you hover on join us

File: package-lock.json [hunk 7]
```diff
@@ -13505,7 +13505,7 @@
     "sinon": {
       "version": "9.0.2",
       "resolved": "https://registry.npmjs.org/sinon/-/sinon-9.0.2.tgz",
-      "integrity": "sha1-uQF+JGM/SxyY37bnhKXwUJ9f2F0=",
+      "integrity": "sha512-0uF8Q/QHkizNUmbK3LRFqx5cpTttEVXudywY9Uwzy8bTfZUhljZ7ARzSxnRHWYWtVTeh4Cw+tTb3iU21FQVO9A==",
```
Comment (seemavora) [inline package-lock.json:13508]: wait why was this file changed?

Comment (seemavora) [reply package-lock.json:13508]: not for sure if it does have to be reverted but if it does use this command:

 git checkout origin/dev package-lock.json

File: src/Components/Navbar/UserNavbar.js [hunk 8]
```diff
@@ -13,71 +14,190 @@ import {
...
-                Student Resources
-              </DropdownToggle>
-              <DropdownMenu right>
-                <DropdownItem href='https://docs.google.com/forms/d/e/1FAIpQ
-                LSfAKfUnblxOZ0r3BjMY6xe_0g2zC7v3OfaadyvF-Ste1eL28A/viewform'>
-                  Microsoft Imagine
+        <NavbarToggler tag='h1'>
+          <ButtonDropdown isOpen={menuIsOpen} toggle={toggler}>
+            <DropdownToggle>
+              <span className='border'></span>
```
Comment (evanugarte) [inline src/Components/Navbar/UserNavbar.js:95]: is this the hamburger menu lol

Comment (nanarboursal) [reply src/Components/Navbar/UserNavbar.js:95]: yea lol would you rather have the picture?

Comment (evanugarte) [reply src/Components/Navbar/UserNavbar.js:95]: nah ur good, this is creative

File: package-lock.json (hunk 7 above)
Comment (evanugarte) [reply package-lock.json:13508]: package-lock changes are good. its means that a package was updated to patch bugs.

File: src/Components/Navbar/navbar.css [hunk 9]
```diff
@@ -21,20 +21,84 @@
...
+  margin-bottom: 5px;
+  padding: 1px;
+  width: 30px;
+  background:	#00008B;
+}
+
+.hamburger-button {
+  background-color: #4e6fa16c !important;
+  border-color: #ffffff00 !important;
+}
```
Comment (IsitaB) [inline src/Components/Navbar/navbar.css:104]: add a blank line to the end of this file
